### PR TITLE
added ctrl+Shift+P menu

### PR DESCRIPTION
**Pull request by ghost**
ctrl+shift+P menu was missing.

### CODE REVIEW

File: content/en/docs/refguide/general/studio-pro-overview.md [hunk 1]
```diff
@@ -239,6 +239,7 @@ The following shortcut keys are assigned to menu actions that can be used in all
 | <kbd>Ctrl</kbd> + <kbd>W</kbd> | Closes the current document. |
 | <kbd>Ctrl</kbd> + <kbd>Shift</kbd> + <kbd>W</kbd> | Closes all document tabs. |
 | <kbd>Delete</kbd> | Deletes the selected element. |
+| <kbd>Ctrl</kbd> + <kbd>Shift</kbd> + <kbd>P</kbd> | Menu quick access search palette. |
```
Comment (MariaShaposhnikova) [inline content/en/docs/refguide/general/studio-pro-overview.md:242]: @mauritselzinga-mx , I've tested it and it does not seem to work for me. Should I open a particular editor first? Or perrhaps you mean Ctrl+Shift+F?

Comment (ghost) [reply content/en/docs/refguide/general/studio-pro-overview.md:242]: Hi, no it is ctrl+shift+p. What version did you test it on? perhaps it's broken in said version?

Comment (MariaShaposhnikova) [reply content/en/docs/refguide/general/studio-pro-overview.md:242]: I am testing on 10.7.0.

**Comment by MariaShaposhnikova** [comment]
I am closing this PR as the issue is not fixed.

**Pull request closed without merging**